scripts/run-framework/agief_experiment/experiment.py

```diff
@@ -431,9 +432,9 @@ def upload_results(self, cloud, compute_node, export_compute):
         if compute_node.remote() and export_compute:
             print "\n --- Upload from exported file on remote machine."
             # remote upload of /output/[prefix] folder
-            cloud.remote_upload_output_s3(compute_node.host_node, self.prefix())
+            cloud.remote_upload_output_s3(compute_node.host_node, self.prefix(), self.no_compress)
         # otherwise, upload it from here
-        else:
+        elif self.no_compress is False:
```

Comment:
Wouldn't we want to upload it from the machine running the python script even if `no_compress == True` ?